### PR TITLE
Feat/nft avatars

### PR DESCRIPTION
**Pull request by rsteubs-bc**
## Description (optional)
Add a concise explanation of the changes.

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

Screen Shot 2022-06-21 at 12 42 03 PM

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -222,5 +222,8 @@
   },
   "resolutions": {
     "qr.js": "https://github.com/defunctzombie/qr.js.git"
+  },
+  "dependencies": {
+    "boring-avatars": "^1.7.0"
```
Comment (plondon) [inline package.json:227]: can you move to wallet-v4-frontend package json?

File: packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/Asset/Asset.tsx [hunk 2]
```diff
@@ -480,7 +485,12 @@ const NftAsset: React.FC<Props> = ({
                                 />
                               ) : (
                                 <Flex alignItems='center' flexDirection='row'>
-                                  <SkeletonCircle height='34px' width='34px' bgColor='grey000' />
+                                  <Avatar
+                                    size={34}
+                                    name={owner.address || ''}
+                                    variant='marble'
+                                    colors={['#0C6CF2', '#5322E5', '#F00699', '#06D6A0', '#121D33']}
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/Asset/Asset.tsx:492]: why hardcoded?

File: packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/Home/TrendingCollectionsTable/name.column.tsx [hunk 3]
```diff
@@ -26,11 +27,21 @@ export const getNameColumn = () => ({
...
+                  alt=''
+                  isVerified={values.safelist_request_status === 'verified'}
+                  src={values.image_url}
+                />
+              ) : (
+                <Avatar
+                  size={24}
+                  name={values.slug || ''}
+                  variant='marble'
+                  colors={['#0C6CF2', '#5322E5', '#F00699', '#06D6A0', '#121D33']}
```
Comment (julianromero-bc) [inline packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/Home/TrendingCollectionsTable/name.column.tsx:41]: This list of strings is so many times duplicated, it needs to have some defined global constant

File: packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/Asset/Asset.tsx (hunk 2 above)
Comment (rsteubs-bc) [reply packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/Asset/Asset.tsx:492]: just some of the colors weren't in constellation.  changing to closest ones

**Review by julianromero-bc — APPROVED** [review]

**Pull request merged**